readme.md

```diff
@@ -135,23 +135,23 @@ $ npm -v
 
 * **Question:** What is `path` and where does it come from?
 
-* **Your Answer:**
+* **Your Answer:**  Path is a module we import and is installed by NPM.
 
 ---
 
 - [ ] Install the [moment](https://www.npmjs.com/package/moment) package
 
 * **Question:** What command can you run to install this package?
 
-* **Your Answer:**
+* **Your Answer:** npm install package_name
 
 ---
 
 - [ ] On your own, use this package in the `index.js` file
 
 * **Question:** Do you need to use a `./` to require the package? Why or why not?
 
-* **Your Answer:**
+* **Your Answer:**  ./ lets node know to look in the same realitive path it is running inside of.
```

Comment:
Correct, and in this case we are not looking for a relative file path but instead an installed module. In that case, we'll look inside of `node_modules/`.